bin/j5activate.cmd

```diff
@@ -3,6 +3,7 @@ if "%1"=="-?" goto usage
 if "%1"=="/?" goto usage
 if "%1"=="-h" goto usage
 if "%1"=="/h" goto usage
+if "%1"=="--python3" goto nolabel
```

Comment:
It would be nice to be able to use this with a version as well

Reply:
@davidfraser You can ... you have to out the --python3 after it (i.e. at %2) and then it works.. a bit non-standard - but workable...

Reply:
I'll do this on a different branch